src/test/java/seedu/uninurse/logic/commands/EditRemarkCommandTest.java

```diff
@@ -0,0 +1,191 @@
+package seedu.uninurse.logic.commands;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static seedu.uninurse.logic.commands.CommandTestUtil.assertCommandFailure;
+import static seedu.uninurse.logic.commands.CommandTestUtil.assertCommandSuccess;
+import static seedu.uninurse.logic.commands.CommandTestUtil.showPersonAtIndex;
+import static seedu.uninurse.logic.commands.EditRemarkCommand.EDIT_REMARK_COMMAND_TYPE;
+import static seedu.uninurse.logic.commands.EditRemarkCommand.MESSAGE_EDIT_DUPLICATE_REMARK;
+import static seedu.uninurse.logic.commands.EditRemarkCommand.MESSAGE_EDIT_REMARK_SUCCESS;
+import static seedu.uninurse.testutil.Assert.assertThrows;
+import static seedu.uninurse.testutil.TypicalIndexes.INDEX_FIRST_ATTRIBUTE;
+import static seedu.uninurse.testutil.TypicalIndexes.INDEX_FIRST_PERSON;
+import static seedu.uninurse.testutil.TypicalIndexes.INDEX_SECOND_ATTRIBUTE;
+import static seedu.uninurse.testutil.TypicalIndexes.INDEX_SECOND_PERSON;
+import static seedu.uninurse.testutil.TypicalPersons.getTypicalUninurseBook;
+
+import org.junit.jupiter.api.Test;
+
+import seedu.uninurse.commons.core.Messages;
+import seedu.uninurse.commons.core.index.Index;
+import seedu.uninurse.model.Model;
+import seedu.uninurse.model.ModelManager;
+import seedu.uninurse.model.UninurseBook;
+import seedu.uninurse.model.UserPrefs;
+import seedu.uninurse.model.person.Patient;
+import seedu.uninurse.model.remark.Remark;
+import seedu.uninurse.testutil.PersonBuilder;
+
+/**
+ * Contains integration tests (interaction with the Model) and unit tests for {@code EditRemarkCommand}.
+ */
+class EditRemarkCommandTest {
+    private static final String REMARK_STUB = "Some remark";
```

Comment:
Same as my other comment in `EditRemarkCommandParserTest`, though this falls under the standardisation issue

Reply:
Same as above.